From EE to Tag or Arm_refer_base to set position:
M2_R_w: 0
M2_R_x: 0
M2_R_y: 1
M2_R_z: 0
M2_x: 0
M2_y: 0
M2_z: 0.126
Reference_position:
- 0.04147865063328958
- -0.0009747523594091701
- 0.0788922767828434
- 1
Reference_rotation:
- -0.13826266849719096
- -0.1382012230239521
- 0.6935305754449016
- 0.6933968541735557

